backend/postgres/schema.sql

```diff
@@ -0,0 +1,57 @@
+CREATE SCHEMA IF NOT EXISTS durabletask;
```

Comment:
I think the schema should be a configurable option. WDYT?

Reply:
Ideally, yes, but I figured this is something we can add later if we get requests for it.